src/constraint/SequentialSampleable.cpp

```diff
@@ -0,0 +1,135 @@
+#include "aikido/constraint/SequentialSampleable.hpp"
+
+#include <dart/common/StlHelpers.hpp>
```

Comment:
Nit: I guess you meant `dart/common/Memory.hpp` for [`dart::common::make_unique`].